applications/nrf_desktop/src/events/led_ready_event.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef _LED_READY_EVENT_H_
+#define _LED_READY_EVENT_H_
+
+/**
+ * @brief LED Ready Event
+ * @defgroup led_ready_event LED Ready Event
+ * @{
+ */
+
+#include "event_manager.h"
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+/** @brief LED ready event used to notify that leds are ready for next step . */
```

Comment:
"next LED stream effect" ?

btw. maybe move this commit before adding application modules connected with LED stream

Reply:
done